source/SharedCrtResourceManager.cpp

```diff
@@ -174,22 +173,22 @@ bool SharedCrtResourceManager::setupLogging(const PlainConfig &config) const
     return true;
 }
 
-void SharedCrtResourceManager::initializeAllocator(const PlainConfig &config)
+void SharedCrtResourceManager::initializeAllocator(const aws_mem_trace_level &configMemTraceLevel)
 {
-    allocator = aws_default_allocator();
```

Comment:
The memory trace allocator **wraps** the allocator being traced and as a result I think you must keep this line.

```
allocator = aws_default_allocator();
```